harambe/__init__.py

```diff
@@ -2,6 +2,18 @@
 from .parser.schemas import Schemas
 from .types import AsyncScraperType, ScrapeResult
 from .utils import PlaywrightUtils
+from harambe.observer import (
+    LocalStorageObserver,
+    LoggingObserver,
+)
+
+
+def save_cookies(page, domain, stage):
+    # Initialize SDK with necessary observers
+    observers = [LocalStorageObserver(domain, stage), LoggingObserver()]
+    sdk = SDK(page, domain=domain, stage=stage, observer=observers)
+    return sdk.save_cookies()
```

Comment:
don't think we need this

Reply:
ah i thought by this sentence `Ensure the cookie / local storage objects are exported by harambe so it can be loaded in other packages, libs, etc.` you need it to be directly accessible from harambe using
`from harambe import save_cookies`
without this it will be accessible only via `sdk.save_cookies`

Reply:
ah apologies for the confusion. 

```
    async def on_save_cookies(self, cookies: List[dict[str, Any]]) -> None:
```

in our code we implement this function. Instead of being a List[dict[str,Any]] it should be a list of List[BrowserCookie] where BrowserCookie has all of the necessary typing

Reply:
is BrowserCookie available in harambe?